.cmake-format

```diff
@@ -0,0 +1,241 @@
+# ----------------------------------
```

Comment:
No, we don't. It's just a proposal. I'm fine with leaving cmake style as it is, but I figured that since we are addressing the code style topic, it might be a good opportunity to have a unified style for make files as well (it's still "code" after all).

Reply:
+1 from me, don't bother with cmake format.